compiler/ballerina-treegen/src/main/java/io/ballerinalang/compiler/internal/treegen/targets/nodevisitor/BaseNodeModifierTarget.java

```diff
@@ -0,0 +1,74 @@
+/*
+ *  Copyright (c) 2025, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
The other classes here also seem to have it, but is "All Rights Reserved" intentional/necessary?

Reply:
No, we can remove that. Let's remove only in the new class right?